src/main/kotlin/com/jetbrains/snakecharm/codeInsight/SmkCodeInsightScope.kt

```diff
@@ -6,15 +6,22 @@ import com.jetbrains.snakecharm.lang.psi.SmkRunSection
 
 enum class SmkCodeInsightScope {
     TOP_LEVEL,
-    RULELIKE_RUN_SECTION;
+    RULELIKE_RUN_SECTION,
+    PEP_SECTION;
 
     fun includes(second: SmkCodeInsightScope) = when (this) {
-         TOP_LEVEL -> second == TOP_LEVEL
-         RULELIKE_RUN_SECTION -> second == TOP_LEVEL || second == RULELIKE_RUN_SECTION
+        TOP_LEVEL -> second == TOP_LEVEL
+        RULELIKE_RUN_SECTION -> second == TOP_LEVEL || second == RULELIKE_RUN_SECTION
+        PEP_SECTION -> second == PEP_SECTION
     }
 
     companion object {
+
+        private fun isPep(anchor: PsiElement): Boolean =
+            anchor.text == "pep"
```

Comment:
According to `text` documentation it should be avoided:

![image](https://user-images.githubusercontent.com/72933/129601506-6fc31e62-bc03-40e4-b7b4-6c5b9f4eefb5.png)

Reply:
Also seems it makes sense only for `PyReferenceExpression` and you could check `PyReferenceExpression.referenceName`

Reply:
https://github.com/JetBrains-Research/snakecharm/pull/424/commits/646025715894f8f53297667ecdbe6c1f4b3f3d62